scripts/apps/workspace/content/constants.ts

```diff
@@ -102,6 +102,23 @@ export const GET_LABEL_MAP = () => ({
     usageterms: gettext('Usage Terms'),
 });
 
+const DEFAULT_STAGES_LABELS = [
```

Comment:
I don't think it would work now if defined on top level right @tomaskikutis ? should be fine if you define it inside that `getLabelForStageIfExists` func

Reply:
you mean `gettext` wouldn't work? `superdesk.pot` seems like it did extract it

Reply:
yep, you're right. You can't call `gettext` from the top level of the code, because top level gets executed before everything else including the loading of translations. Instead, you can define a function and call it when these labels are needed in the UI. UI is only initialized after the translations have been loaded

Reply:
ok I'll change that